### DISCUSSION TITLE
this transaction reverted during simulation

### DISCUSSION THREAD

**Discussion by EduardIshmukhametov** [phantom/docs#208] [Help & Support] [answered]
### Summary

sometimes, when I try to borrow here https://texture.finance/lendy/borrow
I got error "this transaction reverted during simulation" but after few seconds its ok
![image](https://github.com/phantom/docs/assets/45166336/dfa938c6-dbc9-4444-8877-8dcccc6ff942)
![image](https://github.com/phantom/docs/assets/45166336/7cb8367a-5e95-4714-9655-a550f806b998)

### Steps to Reproduce

1. go to https://texture.finance/lendy/borrow
2. borrow something

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
@EduardIshmukhametov Thanks for writing in!

We have had some issues with our transaction simulation over the past few days. Are you still seeing this error?

**Reply by TannerIsBender** [reply]
I'm encountering an error when trying to revoke MintAuthority in my local development environment. Strangely, the same operation works fine in the production environment, which hasn't changed. Both environments are using the same package versions.

When I add this instruction to the transaction, I receive the following error:

"This transaction reverted during simulation. Funds may be lost if submitted."

Here's the relevant code snippet:


```
createNewTokenTransaction.add(
  createSetAuthorityInstruction(
    mintKeypair.publicKey,
    publicKey,
    AuthorityType.MintTokens,
    null // Revoking mint authority
  )
);
```
Any insights into why this might be happening or how to resolve it would be greatly appreciated.
